modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/jaxrs/exception/mapper/DiscountMaxPriceValueExceptionMapper.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.headless.commerce.admin.pricing.internal.jaxrs.exception.mapper;
+
+import com.liferay.commerce.constants.CommerceExceptionMapperConstants;
+import com.liferay.commerce.discount.exception.CommerceDiscountMaxPriceValueException;
+import com.liferay.headless.commerce.core.exception.mapper.BaseExceptionMapper;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Christian Chiappa
+ */
+@Component(
+	enabled = false,
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Headless.Commerce.Admin.Pricing)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Headless.Commerce.Admin.Pricing.DiscountMaxPriceValueExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+@Provider
+public class DiscountMaxPriceValueExceptionMapper
+	extends BaseExceptionMapper<CommerceDiscountMaxPriceValueException> {
+
+	@Override
+	public String getErrorDescription() {
+		return CommerceExceptionMapperConstants.
+			MAX_PRICE_VALUE_EXCEPTION_MESSAGE;
```

Comment:
Same here: is this constant really needed?

Reply:
https://github.com/brianchandotcom/liferay-portal/pull/117940#issuecomment-1137506641 Brian rejected this pr because the error message was duplicated for the two exception mapper class. So I've created a constant

Reply:
Ok, thank you.